app/components/UnderlinedText.tsx

```diff
@@ -1,11 +1,11 @@
-import React from "react";
+import { FC } from "react";
```

Comment:
This isn't required... Could you take some time today to research this? If my understanding is correct, it comes from not restructuring your props, which you are doing now. You might be able to remove it altogether without any errors.